src/wallet.js

```diff
@@ -869,17 +870,21 @@ const wallet = {
    * @inner
    */
   markWalletAsStarted() {
-    return storage.setItem('wallet:started', true);
+    return StorageProxy.getStorage().setItem('wallet:started', true);
   },
 
   /**
    * Subscribe to receive updates from an address in the websocket
    *
    * @param {string} address
    */
-  subscribeAddress(address) {
+  subscribeAddress(address, connection = null) {
     const msg = JSON.stringify({'type': 'subscribe_address', 'address': address});
-    WebSocketHandler.sendMessage(msg);
+    WebSocketHandler.ws.sendMessage(msg);
```

Comment:
It seems we introduced an incompatibility because of the `.ws.` part. I think we shouldn't do it otherwise we will have to refactor our wallets as well.

Reply:
Yes, the only way to keep the compatibility is if we keep the `WebSocketHandler` as it is. But since we are already breaking compatibility with Storage in the wallets (unless we think about a different design), I think I can refactor the wallet desktop and mobile with these changes. Or do you thinks it's risky?

Reply:
I don't think it is risky, but I think it should be simple. As we will refactor the wallets, why don't we just remove the `WebSockerHandler.js` and directly use the new connection class?

Reply:
The `WebSocketHandler` wasn't removed because it's not used only in the wallets but in our old code of the lib as well. We agreed not to refactor that code right now.

Reply:
We could have on `WebSocketHandler`, so no refactor on `wallet.js` is needed for this.
```js
sendMessage = msg => {
    return this.ws.sendMessage(msg);
}
```

Reply:
I don't mind changing this on the lib, as long as the wallet code does not change. If I create this `sendMessage`, I would also need to override the `emit` method from `WebSocketHandler` because we have some calls like:

`WebSocketHandler.ws.emit('addresses_loaded', result);`

That's why I prefer keeping the refactor as it is.

Reply:
I agree. As lon as the wallet code does not change, it is good to go.